src/UI/Component/ViewControl/Factory.php

```diff
@@ -94,4 +94,39 @@ public function section(Button $previous_action, \ILIAS\UI\Component\Component $
 	 */
 	public function sortation(array $options);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      Pagination allows structured data being displayed in chunks by
+	 *      limiting the number of entries shown. It provides the user with
+	 *      controls to leaf through the chunks of entries.
+	 *   composition: >
+	 *      Pagination is a collection of shy-buttons to access distinct chunks
+	 *      of data, framed by additional shy-buttons with chevron left/right glyphs.
+	 *   effect: >
+	 *      A click on an chunk-option will change the offset of the displayed data-list,
+	 *      thus displaying the respective chunk of entries.
+	 *      The active option is rendered as an unavailable shy-button.
+	 *      Clicking the chevron-buttons, the previous (respectively: the next)
+	 *      chunk of entries is being displayed.
+	 *      If a previous/next chunk is not available, the chevron-button is
+	 *      rendered unavailable.
+	 *      If the pagination is used with a maximum of chunk-options to be shown,
+	 *      both first and last options are always displayed.
+	 *
+	 * rules:
+	 *   usage:
+	 *      1: A Pagination MUST NOT only be used for structured data, like tables and lists.
```

Comment:
To much "NOT" in this line: "A Pagination MUST -NOT- only be used for structured data, like tables and lists."